src/components/SvgIcon.tsx

```diff
@@ -0,0 +1,15 @@
+import { FC } from 'react';
+import { Icon } from '@chakra-ui/react';
+import { customIcons } from '@styles/icons';
+
+interface SvgIconProps {
+  name: keyof typeof customIcons;
+  size?: number;
+}
+
+const SvgIcon: FC<SvgIconProps> = ({ name, size = 16 }) => {
+  const TargetIcon = customIcons[name];
+  return <Icon as={TargetIcon} boxSize={size} />;
+};
+
+export default SvgIcon;
```

Comment:
```suggestion
import { FC } from 'react';
import { Icon, IconProps } from '@chakra-ui/react';
import { customIcons } from '@styles/icons';

interface SvgIconProps extends IconProps {
  name: keyof typeof customIcons;
}

const SvgIcon: FC<SvgIconProps> = ({ name, boxSize = 16, ...restProps }) => {
  const TargetIcon = customIcons[name];
  return <Icon as={TargetIcon} boxSize={boxSize} {...restProps} />;
};

export default SvgIcon;
```

이 방법은 어떠신가요?! 차크라에서 제공하는 다른 props를 바로 활용할 수 있는 방법입니다.

Reply:
차크라가 제공하는 IconProps가 있었군요! 감사합니다 👍  
[e8235b5](https://github.com/mash-up-kr/moit-web/pull/15/commits/e8235b5012bb68225a4f17c5e623f1eb933236c5) 커밋에서 적용하였습니다

Reply:
차크라가 제공하는 props를 활용해서 커스텀이 가능합니다! 저는 BoxProps를 제일 많이 썼던 것 같아요 ㅎㅎ 반영해주셔서 감사합니다!